Sources/MeiliSearch/QueryParameters/TasksQuery.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+public class TasksQuery: Queryable {
+  private var from: Int?
+  private var limit: Int?
+  private var next: Int?
+  private var types: [String]
+  var indexUid: [String]
+  private var status: [String]
+
+  init(limit: Int? = nil, from: Int? = nil, next: Int? = nil, status: [String]? = nil, types: [String]? = nil, indexUid: [String]? = nil) {
+    self.from = from
+    self.limit = limit
+    self.next = next
+    self.status = status ?? []
+    self.types = types ?? []
+    self.indexUid = indexUid ?? []
+  }
```

Comment:
```suggestion
  init(limit: Int? = nil, from: Int? = nil, next: Int? = nil, status: [String]? = [], types: [String]? = [], indexUid: [String]? = []) {
    self.from = from
    self.limit = limit
    self.next = next
    self.status = status
    self.types = types
    self.indexUid = indexUid
  }
```

would this work?

Reply:
It works if I do some changes in the `buildQuery()` method, but I think its better with your suggestion!

Reply:
update: it does not work, because Meilisearch engine does not support `status=` or `type=` in the query string.

a validation like this will not work `status != nil ? status?.joined(separator: ",") : nil,` because since status is optional I will have to unwrap it before using it.